src/App.js

```diff
@@ -8,8 +8,8 @@ import {menuData} from './Components/Menu/menuData';
 import './Components/Header/Banner.css';
 import './Components/Menu/Menu.css';
 import './Components/DynamicContent/Search/Search.css';
-import './Components/DynamicContent/Config/Config.css';
 import './Components/DynamicContent/Asset/AssetManager.css';
+import './Components/DynamicContent/PromotionBuilder/PromotionBuilder.css';
```

Comment:
all css should not be imported into App. Each Component should import its own CSS.